<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TFDialog</class>
 <widget class="QDialog" name="TFDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>824</width>
    <height>808</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Transfer Function</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>801</width>
     <height>301</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tf1DTab">
    <attribute name="title">
     <string>&amp;1D Transfer Function</string>
    </attribute>
    <widget class="QGraphicsView" name="color1DView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>768</width>
       <height>30</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>768</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>768</width>
       <height>30</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tf2DTab">
    <attribute name="title">
     <string>&amp;2D Transfer Function</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>770</y>
     <width>681</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="dialogButtonLayout">
    <item>
     <widget class="QPushButton" name="applyButton">
      <property name="text">
       <string>Apply</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="dialogButtonSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="closeButton">
      <property name="text">
       <string>Close</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>320</y>
     <width>643</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="widgetButtonLayout">
    <item>
     <widget class="QPushButton" name="colorButton">
      <property name="text">
       <string>Color</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pyramidButton">
      <property name="text">
       <string>Pyramid</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="gaussianButton">
      <property name="text">
       <string>Gaussian</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="customButton">
      <property name="text">
       <string>Custom</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="skipRangeButton">
      <property name="text">
       <string>Skip Range</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="deleteButton">
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="undoButton">
      <property name="text">
       <string>Undo</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>165</width>
     <height>194</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="presetColorsGroupBox">
      <property name="title">
       <string>Preset Colors</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QComboBox" name="presetColorsBox">
         <item>
          <property name="text">
           <string>Bright Colors</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Hue Gradient</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Grayscale</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>White</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Red</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Green</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Blue</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Cool to Warm</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="presetAlphaGroupBox">
      <property name="title">
       <string>Preset Alpha</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QComboBox" name="presetAlphaBox">
         <item>
          <property name="text">
           <string>Ascending</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Descending</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Opaque</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="mouseLabel">
      <property name="text">
       <string>Mouse:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="pinLabel">
      <property name="text">
       <string>Pin:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>TFDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>771</x>
     <y>583</y>
    </hint>
    <hint type="destinationlabel">
     <x>411</x>
     <y>307</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
